.travis.yml

```diff
@@ -16,7 +15,7 @@ env:
       - BUILD_DEPENDS="Cython"
       - TEST_DEPENDS=
 
-matrix:
+jobs:
   exclude:
     # Exclude the default Python 3.5 build
     - python: 3.5
```

Comment:
The python version is 3.5 above but 3.5 is specified as "exclude" here. Is it the reason that making the build skipped?

Reply:
Actually this link might gives the real reason: https://docs.travis-ci.com/user/build-matrix/#explicitly-included-jobs-with-only-one-element-in-the-build-matrix

Reply:
I am not sure what this does.